app/src/main/res/values/strings.xml

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 <resources>
     <string name="os_licence_version_text">Version:</string>
     <string name="os_licence_licence_text">Licence:</string>
-    <string name="os_licences">Licence info</string>
+    <string name="title_nav_os_licences">Licence info</string>
```

Comment:
Put this String with 
So in future contributor can easily able to find the string and make changes if needed.